### PR TITLE
Add legend or hover tooltip for filters (#3404)

### PR DESCRIPTION
**Pull request by tablatronix**
Adds tooltips to filter checkboxes

### CODE REVIEW

File: wled00/data/index.css [hunk 1]
```diff
@@ -461,7 +461,7 @@ button {
 
 .filter {
 	z-index: 1;
-	overflow: hidden;
+	overflow: visible;
```
Comment (blazoncek) [inline wled00/data/index.css:464]: I am not sure this change is appropriate as we do not want filter bar to wrap into two rows.

Comment (tablatronix) [reply wled00/data/index.css:464]: Yeah I didn't this it was a good idea, but saw nothing break, alternative is to add another wrapper and change its size to be tall enough for the tooltips maybe but might have to then account for the top bar option toggle

Comment (blazoncek) [reply wled00/data/index.css:464]: Screenshot 2023-10-14 at 18 36 24
Not good.

**Comment by blazoncek** [comment]
Correctly implemented in 0_15 branch.

**Pull request closed without merging**